cli/src/nonce.rs

```diff
@@ -656,6 +684,40 @@ pub fn process_withdraw_from_nonce_account(
     }
 }
 
+pub(crate) fn process_upgrade_nonce_account(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    nonce_account: Pubkey,
+    memo: Option<&String>,
+) -> ProcessResult {
+    let latest_blockhash = rpc_client.get_latest_blockhash()?;
+    let ixs = vec![upgrade_nonce_account(nonce_account)].with_memo(memo);
+    // TODO what signers?
```

Comment:
Is this comment still needed?

Reply:
Yeah, this code is still in WIP until https://github.com/solana-labs/solana/pull/25788 is merged.

Reply:
removed